window.go

```diff
@@ -22,21 +22,26 @@ import (
 	"github.com/winlabs/gowin32/wrappers"
 )
 
-func GetWindowProcessId(hwnd syscall.Handle) syscall.Handle {
+func GetWindowProcessID(hwnd syscall.Handle) uint {
 	var pid uint32
-	wrappers.GetWindowThreadProcessId(hwnd, &pid)
-	return syscall.Handle(pid)
+	wrappers.GetWindowThreadProcessID(hwnd, &pid)
+	return uint(pid)
 }
 
 func GetWindowText(hwnd syscall.Handle) (string, error) {
-	buf := make([]uint16, 512)
-	if _, err := wrappers.GetWindowText(hwnd, &buf[0], len(buf)); err != nil {
+	l, err := wrappers.GetWindowTextLength(hwnd)
+	if err != nil {
+		return "", err
```

Comment:
When returning an error from an API call, we should use `NewWindowsError` to wrap it and specify which API function failed.